configure_cd.md

```diff
@@ -1,18 +0,0 @@
-# Configure a Continuous Deployment (CD)
```

Comment:
these two files (this and the next) were duplicated and got merged with the other one to simplify the reading